api/api/clubs/migrations/0001_initial.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.0.5 on 2022-07-08 10:39
```

Comment:
remove all the migration files. Once we are confident about all the db schemas, we'll push migrations together in one go.